venues/ICLR.cc/2019/webfield/bidWebfield.js

```diff
@@ -360,7 +360,7 @@ function renderContent(validNotes, tagInvitations, metadataNotesMap) {
       container: '#allPapers',
       search: {
         enabled: true,
-        localSearch: false,
+        localSearch: true,
```

Comment:
Bids can use local search because we load all the notes, right?

Reply:
ah right. yes, makes sense